libafl_qemu/src/qemu/qemu_config.rs

```diff
@@ -0,0 +1,404 @@
+use core::{
+    fmt,
+    fmt::{Display, Formatter},
+};
+use std::{
+    path::{Path, PathBuf},
+    sync::OnceLock,
+};
+
+use getset::Getters;
+use libafl_derive;
+use strum_macros;
+use typed_builder::TypedBuilder;
+
+use crate::{Qemu, QemuInitError};
+
+pub(super) static QEMU_CONFIG: OnceLock<QemuConfig> = OnceLock::new();
+
+#[cfg(emulation_mode = "systemmode")]
+#[allow(non_camel_case_types)]
+#[derive(Debug, strum_macros::Display, Clone)]
+#[strum(prefix = "-accel ")]
+pub enum Accelerator {
+    kvm,
```

Comment:
Why not camel case?

Reply:
FWIW you can use `#[strum(serialize_all = "snake_case")]` to make this boi lowercase as string

Reply:
Good point, fixed thanks!